en/docs/learn/configuring-tenant-based-adaptive-authentication.md

```diff
@@ -76,13 +75,12 @@ credentials (basic authentication) to access a resource.
     with the appended tenant domain (i.e., chris@abc.com).  
     ![pickup-sign-in-chris](../assets/img/tutorials/pickup-sign-in-chris.png)  
     Note that you are prompted for harware key authentication because
-    **abc.com** is a whitelisted tenant domain.
+    **abc.com** belongs to the allowlist of the tenant domain.
```

Comment:
belongs to the tenant domains allowlist

Reply:
Rephrased this as "**abc.com** is a domain that belongs to the allowlist".